### ISSUE TITLE
Duplicate mapping?

### ISSUE THREAD

**Issue by vicn1222** [leoliu0/cik-cusip-mapping#5]
Hi,

Excellent job! Thank you.

I have some questions.

The map between cik and cusip is not 1 to 1. It is many to many. Is this correct?

For instance, cusip G1117K maps to 2 ciks (1769484, 1725206), while cik 1172939 maps to 6 cusips (49900N, 024855, 024857, 027858, 024884, 024856).

Can you explains why?

Thanks!

**Comment by Jon-Nie** [comment]
As far as I know, ciks are set up once by the sec and always belong to one specific company. 
So for example the cik 320193 will always map to Apple Inc. and if Apple goes bankrupt, the cik will not be assigned to another company.
Cusips however are assigned by Standard & Poors and can change when a major event happens (I do not know what even though). For reference: https://www.finra.org/filing-reporting/market-transparency-reporting/uniform-practice-code-upc/faq Question 7

To sum up, cusips change over time and map to one cik/company first and, if changed, can eventually map to another company/cik.

**Comment by leoliu0** [comment]
Thank you Jon-Nie for explaining. Yes, CIK is a company identifier (more specifically, entity identifier) and CUSIP is a security identifier. One company can have multiple issues (equity with different classes, bonds etc...). So CIK to multiple CUSIPs is not uncommon. CUSIP to multiple CIKs are relatively less common but can happen when a major corporate event happen such as M&A.
